src/backend/distributed/operations/shard_transfer.c

```diff
@@ -996,6 +996,20 @@ ReplicateColocatedShardPlacement(int64 shardId, char *sourceNodeName,
 		return;
 	}
 
+	WorkerNode *sourceNode = FindWorkerNode(sourceNodeName, sourceNodePort);
+	WorkerNode *targetNode = FindWorkerNode(targetNodeName, targetNodePort);
+
+	Oid relationId = RelationIdForShard(shardId);
+	PlacementUpdateEvent *placementUpdateEvent = palloc0(
+		sizeof(PlacementUpdateEvent));
+	placementUpdateEvent->updateType = PLACEMENT_UPDATE_COPY;
```

Comment:
would anything break if we added the operation name to the output?

Reply:
No, it should be fine to add some columns to the output. So, did that now.